google/datalab/stackdriver/commands/_monitoring.py

```diff
@@ -106,3 +107,10 @@ def _render_dataframe(dataframe):
   fields = dataframe.columns.tolist()
   return IPython.core.display.HTML(
       google.datalab.utils.commands.HtmlBuilder.render_table(data, fields))
+
+
+def _make_context(project_id):
+  context = google.datalab.Context.default()
```

Comment:
This way you're setting the project id for the global context object, which will affect all other operations. Instead, what you want is copy the default context object into a new one, and change that one's project and use it.